.gitignore

```diff
@@ -6,3 +6,4 @@ rebusurance.zip
 config.js
 .parcel-cache
 dist/
+style.json
```

Comment:
The script doesn’t hard-code this path – it seems to just be a path you happen to be using on the command line. If anything, dist/ (which is already ignored) would be a better home for this JSON file.

Reply:
Would it be better for the script to dump a file or dump to stdout?

Reply:
It’s a pretty big file, so dumping to a file would make sense. Perhaps this script could take an output path as a command-line option and fall back to stdout if there isn’t one.

Reply:
The script already uses the command-line option of a varargs list of language codes. Is there a more sophisticated way to handle arbitrary command-line options without reinventing the wheel?

Reply:
I'd suggest [`commander`]:

```
import { Command } from "commander";
const program = new Command();
program
  .option("-l, --locales <locales...>", "language codes")
  .option("-o, --outfile <file>", "output file", "-");
program.parse(process.argv);
console.log("%o", program.opts());
```

```
$ node ./scripts/generate_style.js --help 
Usage: generate_style [options]

Options:
  -l, --locales <locales...>  language codes
  -o, --outfile <file>        output file (default: "-")
  -h, --help                  display help for command
$ node ./scripts/generate_style.js -l it vi 
{ outfile: '-', locales: [ 'it', 'vi', [length]: 2 ] }
```

[`commander`]: https://www.npmjs.com/package/commander

Reply:
Added this in a1b136e.  Thanks for the suggestion of commander, it was really easy to work with!

Reply:
Appears to have forgotten dependency change?

Reply:
What do you mean?

Reply:
`npm i commander`